server/controllers/chefsController.js

```diff
@@ -0,0 +1,11 @@
+const connection = require("../dbConnection");
```

Comment:
why are the back-end files being added to the PR meant for front-end

Reply:
The issue requested that I make one to get the user information.  We didn't need one for the initial login/sign up since you were already returning the user information.  However I did make one to get the user information when a cookie existed.  Let me know if you already have a route like this and I can either update my branch to match yours for that part or you can remove what I have and update the front end route to match yours.